addons/mail/models/mail_message.py

```diff
@@ -959,6 +968,7 @@ def message_format(self, format_reply=True):
                 'subtype_description': message_sudo.subtype_id.description,
                 'is_notification': vals['message_type'] == 'user_notification',
                 'recipients': [{'id': p.id, 'name': p.name} for p in message_sudo.partner_ids],
+                'user_follower_id': followed_refs.get((vals['model'], vals['res_id']), 0),
```

Comment:
shouldn't we write False instead of zero ? (a bit more "Odoo style")

Reply:
Yes it works, thanks